nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java

```diff
@@ -292,7 +304,11 @@ HdfsResources resetHDFSResources(String configResources, ProcessContext context)
                 ugi = SecurityUtil.loginKerberos(config, principal, keyTab);
                 fs = getFileSystemAsUser(config, ugi);
                 lastKerberosReloginTime = System.currentTimeMillis() / 1000;
-            } else {
+            } else if (context.getProperty(REMOTE_USER).isSet()){
```

Comment:
This might be more of a stylistic thing, but we could probably fold these together....

```
else {
  config.set("ipc.client.fallback-to-simple-auth-allowed", "true");
  config.set("hadoop.security.authentication", "simple");		                  
  if (context.getProperty(REMOTE_USER).isSet()) {
    ugi = UserGroupInformation.createRemoteUser(context.getProperty(REMOTE_USER).evaluateAttributeExpressions().getValue());
  } else {
    ugi = SecurityUtil.loginSimple(config);
  }
  fs = getFileSystemAsUser(config, ugi);
}
```